symfony/client/src/orangehrmPimPlugin/components/SaveMembership.vue

```diff
@@ -0,0 +1,183 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-horizontal-padding orangehrm-vertical-padding">
+    <oxd-text tag="h6" class="orangehrm-main-title">
+      Add Membership
+    </oxd-text>
+    <oxd-divider />
+    <oxd-form :loading="isLoading" @submitValid="onSave">
+      <oxd-form-row>
+        <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+          <oxd-grid-item>
+            <oxd-input-field
+              type="select"
+              label="Membership"
+              v-model="membership.membership"
+              :options="memberships"
+              required
+            />
+          </oxd-grid-item>
+          <oxd-grid-item>
+            <oxd-input-field
+              type="select"
+              label="Subscription Paid By"
+              v-model="membership.subscriptionPaidBy"
+              :options="paidBy"
+            />
+          </oxd-grid-item>
+          <oxd-grid-item>
+            <oxd-input-field
+              label="Subscription Amount"
+              v-model="membership.subscriptionFee"
+              :rules="rules.subscriptionFee"
+            />
+          </oxd-grid-item>
+          <oxd-grid-item>
+            <oxd-input-field
+              type="select"
+              label="Currency"
+              v-model="membership.currencyType"
+              :options="currencies"
+            />
+          </oxd-grid-item>
+          <oxd-grid-item>
+            <oxd-input-field
+              label="Subscription Commence Date"
```

Comment:
Please rebase and use `date-input`. [refer here](https://github.com/orangehrm/orangehrm/blob/5.0/symfony/client/src/orangehrmPimPlugin/components/SaveLicense.vue#L57)